transport/src/main/java/io/netty/channel/DefaultChannelHandlerContext.java

```diff
@@ -76,6 +81,22 @@
         this.handler = handler;
     }
 
+    private static void failRemoved(DefaultChannelHandlerContext ctx, ChannelPromise promise) {
+        promise.setFailure(newRemovedException(ctx, null));
+    }
+
+    private void fireHandlerRemoved() {
+        fireHandlerRemoved(null);
+    }
```

Comment:
This might deserve a different name: `ChannelHandler` has a method `handlerRemoved` which make it sound like this method fires a handler removed event, not an exception.

Reply:
good point.

Reply:
Are you still interested in changing the name?

Reply:
yep...